src/fastmcp/server/dependencies.py

```
@@ -760,7 +787,33 @@ async def __aenter__(self) -> Context:
+            return
+
+        token_key = docket.key(f"fastmcp:task:{session_id}:{task_id}:access_token")
+        try:
+            async with docket.redis() as redis:
+                token_data = await redis.get(token_key)
+            if token_data is not None:
+                restored = AccessToken.model_validate_json(token_data)
+                self._access_token_cv_token = _task_access_token.set(restored)
+        except Exception:
+            # Don't let token restoration failures break task execution
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)